src/app/src/components/FilterSidebar.jsx

```diff
@@ -95,15 +103,27 @@ class FilterSidebar extends Component {
                 </p>
             </div>);
 
-        const activeTabIndex = filterSidebarTabs
+        const orderedTabsForSidebar = vectorTileFeatureIsActive
+            ? filterSidebarTabs.slice().reverse()
+            : filterSidebarTabs;
+
+        const activeTabIndex = orderedTabsForSidebar
             .findIndex(({ tab }) => tab === activeFilterSidebarTab);
 
         const handleTabChange = (_, value) => {
-            const changeTab = [
-                makeGuideTabActive,
-                makeSearchTabActive,
-                makeFacilitiesTabActive,
-            ][value];
+            const changeTabDictionary = vectorTileFeatureIsActive
```

Comment:
Consider `changeTabList` to reflect the arrayness.

Reply:
Good catch -- I'll rename before merging.